src/main/java/com/googlecode/zohhak/api/backend/ParameterCalculatorProvider.java

```diff
@@ -0,0 +1,16 @@
+package com.googlecode.zohhak.api.backend;
+
+import com.googlecode.zohhak.internal.IoCContainer;
+import com.googlecode.zohhak.internal.coercing.DefaultParameterCoercerFactory;
+
+public class ParameterCalculatorProvider {
+
+    public ParameterCalculator getExecutor() {
+        return getExecutor(new DefaultParameterCoercerFactory());
+    }
+
+    public ParameterCalculator getExecutor(ParameterCoercerFactory parameterCoercerFactory) {
```

Comment:
that's the only entry point to the new backend api. but backend cannot be instantiated using the public api. there is no `ParameterCoercerFactory` implementation in public api